website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,223 @@
+
+- General Settings:
+    - Name: owncloud (or owncloud-desktop, owncloud-android, etc.)
+- Protocol Settings:
+    - Signing Key: select the signing key you wish to use
+- Advanced Protocol Settings:
+    - Scopes: email, offline_access, openid, profile
+
+The following settings are different for the different ownCloud applications:
+
+- Web UI
+    - Protocol Settings:
+        - Client ID: use generated authentik value (or customize if you wish)
+        - Client Secret: use generated authentik value (or customize if you wish)
+        - Redirect URIs: `https://owncloud.company/apps/openidconnect/redirect`
+- Desktop Application:
+    - Protocol Settings:
+        - Client ID: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        - Client Secret: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        - Redirect URI:
+            ```
+            http://localhost:\d+
+            http://127.0.0.1:\d+
+            ```
+- Android Application:
+    - Protocol Settings:
+        - Client ID: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        - Client Secret: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        - Redirect URI: `oc://android.owncloud.com`
+- iOS Application:
+    - Protocol Settings:
+        - Client ID: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        - Client Secret: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        - Redirect URI: `oc://ios.owncloud.com`
+
+##### Applications
+
+#### ownCloud
+
+In the authentik Admin Interface, go to **Applications -> Applications**, and create a new Application with the following settings for each of the providers defined above:
+
+- Name: owncloud, owncloud-desktop, owncloud-android, or owncloud-ios
+- Slug: same as name
+- Provider: one of the providers you created in the previous section
+- UI Settings:
+    - Launch URL: You can set this to `blank://blank` to prevent the application from being listed on the authentik
+      home page. This may be useful for the desktop, android and ios applications, since you will not be able
+      to navigate directly to those applications.
+      :::note
+      Sometimes this field glitches out when creating the application, you may need to save the application and then
+      edit it to fill this field.
+      :::
+
+##### Service Discovery
+
+In order for the ownCloud Applications to be able to login via OIDC, your reverse proxy will have to be configured to rewrite
```

Comment:
```suggestion
In order for the ownCloud Applications to be able to login via OIDC, you need to configure your reverse proxy to rewrite
```

Reply:
ideally, we would say "... configure your reverse proxy to rewrite the <??> file/setting/parm from:"

In other words, where is this setting? What file?